docs/pages/en-us/Features.md

```diff
@@ -22,7 +22,7 @@ User and system audit events are defined per application.
 The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis and across multiple tenants
 
 ## Auto-Generated Client Library
-The service includes an auto-generated client-side library for type safety.  This client library also makes it easier to send user and system audit events to Elasticsearch.
+The service includes an auto-generated client-side library for type safety.  This client library also makes it easier to send user and system audit events to the Audit Web Service or Elasticsearch.
```

Comment:
I would suggest changing this to "This client library also makes it easier to send user and system audit events to Elasticsearch directly or via the Audit Web Service."